### PR TITLE
A pre-build helloworld image written in Go

### PR DESCRIPTION
**Pull request by luebken**
This makes helloworld use very simple and small prebuild image written in Golang. It handles SIGTERM so `swarm stop` want go to a `failed status`. In addition we can have some nice HTML output to include some content.

It will be automatically build by the Docker hub: https://registry.hub.docker.com/u/giantswarm/helloworld/

Fixes: https://github.com/giantswarm/docs/issues/5

TODOs:

* [x] Get a dev LGTM
* [x] Have some nice HTML
* [x] Adapt https://github.com/giantswarm/docs-content/blob/master/content/guides/annotated-helloworld.md

### CODE REVIEW

**Comment by luebken** [comment]
RFR @denderello @marians

**Comment by marians** [comment]
Looks good to me! HTML improvements can be done in a next step. 11 MB seems to be an adequate image size. :)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	echo Building for $(GOOS)/$(GOARCH)
+	docker run \
+	    --rm \
+	    -it \
+	    -v $(shell pwd):/usr/code \
+	    -e GOPATH=/usr/code/.gobuild \
+	    -e GOOS=$(GOOS) \
+	    -e GOARCH=$(GOARCH) \
+	    -w /usr/code \
+	    golang:1.3.1-cross \
```
Comment (denderello) [inline Makefile]: We can use some 1.4 version for this. This is what we decided for all projects recently.

**Comment by denderello** [comment]
Besides the Go version LGTM

**Pull request merged**